aws-cloud-integrations.md

```diff
@@ -442,6 +442,63 @@ You can also check query history to see if any queries are failing:
 	
 <img width="1792" alt="Screen Shot 2020-12-06 at 9 43 50 PM" src="https://user-images.githubusercontent.com/453512/101319633-24ef5500-3817-11eb-9f87-55a903428936.png">
 
+### Common Athena errors.
+
+#### Incorrect bucket in IAM Policy.
+
+* **Symptom:** 
+   A similar error to this will be shown on the diagnostics page under "Pricing Sources" on the "Diagnostics" page. You can search for the <Athena Query ID> in the Athena "Recent queries" dashboard to find additional info about the error.
```

Comment:
I'm unsure how "\<Athena Query ID\>" will render in Markdown. You may need to use the escape character "\\" in front of the carrots. Or you can use the code blocks "`".